src/pages/DeleteReservation.js

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchReservations, deleteReservation } from '../redux/delete-reservation/deleteReservationSlice';
+import '../style/DeleteReservation.css';
+
+const DeleteReservation = () => {
+  const dispatch = useDispatch();
+  const reservations = useSelector((state) => state.reservationsReducer.reservations);
+  // const reservations = useSelector((state) => {
+  //   console.log('State:', state);
+  //   console.log('Reservations:', state.reservations);
+  //   return state.reservations;
+  // });
+
+  useEffect(() => {
+    dispatch(fetchReservations);
+  }, [dispatch]);
+
```

Comment:
- [ ] Well done for implementing the use effect to fetch reservations data, However, Ensure the `fetchReservations` action is called properly. 
`Hint:` 
```
useEffect(() => {
    dispatch(fetchReservations());
  }, [dispatch]);
```